src/lib/execution_plan_test.rs

```diff
@@ -414,6 +414,49 @@ cd -"#
     assert!(task.env.is_none());
 }
 
+#[test]
+#[cfg(target_os = "linux")]
+fn create_workspace_with_cargo_config() {
```

Comment:
you already build a dummy workspace2 for testing which is great.
can we use it also to test crate_target_triple directly? having some test in crate_info_test.rs?
it would also work for all platforms, not just linux which is even better.

Reply:
Done.